addons/project/static/src/views/burndown_chart/burndown_chart_model.js

```diff
@@ -1,14 +1,78 @@
 /** @odoo-module **/
 
 import { GraphModel } from "@web/views/graph/graph_model";
+import { sortBy } from "@web/core/utils/arrays";
+
+export const SEP = " / ";
 
 export class BurndownChartModel extends GraphModel {
+    /**
+     * @override
+     */
+    setup(params) {
+        this.stageSeq = {};
+        super.setup(params);
+    }
+
+    /**
+     * Fetch the sequence of each stage in the project. This function alters this.stageSeq
+     * @protected
+     * @param {Object} searchParams
+     */
+    async _fetchStageSeq(searchParams) {
+        const data = await this.orm.webSearchRead(
+            "project.task.type",
+            [["project_ids", "in", searchParams.context.active_id]],
+            ["name", "sequence", "id"]
+        );
+
+        const dataClean = {};
+        data.records.forEach((obj) => {
+            dataClean[obj.id] = { sequence: obj.sequence, name: obj.name };
+        });
+        return dataClean;
+    }
+
+    /**
+     * @param {SearchParams} searchParams
+     */
+    async load(searchParams) {
+        this.searchParams = searchParams;
+        // if the stage_seq wasn't given by the action (for exemple if the page is simply reloaded)
+        if (!this.searchParams.context.stage_seq) {
+            this.stageSeq = await this._fetchStageSeq(searchParams);
```

Comment:
`this.stageInfo`, `this.stageData`, or even better `stageSequenceAndNamePerId`.
Point being that, when I read your code, I immediately understand what your doing when you write `this.stageSeq[blabla]` :zap:

Reply:
Done